testcases/kernel/syscalls/lchown/lchown01.c

```diff
@@ -108,15 +108,15 @@ int main(int argc, char *argv[])
 					 SFILE, TEST_ERRNO);
 			}
 
-			if (user_id == -1) {
+			if ((int)user_id == -1) {
 				if (i > 0)
 					user_id =
 					    test_cases[i - 1].user_id;
 				else
 					user_id = geteuid();
 			}
 
-			if (group_id == -1) {
+			if ((int)group_id == -1) {
```

Comment:
I guess that it would be cleaner to use int for the user_id and group_id instead of the uid_t and gid_t here.

Reply:
If `user_id` and `group_id` are changed to `int`, I think this will produce more warnings at line 131 and 132 where they are compared against `uid_t` and `gid_t`. What do you suggest?

Reply:
Email Response from Cyril Hrubis <chrubis@suse.cz>:
""Well when change of the type produces warnings at different places of
the code it's probably better just to add casts and leave it that way.""